.changeset/wild-melons-study.md

```diff
@@ -0,0 +1,29 @@
+---
+"cojson-core-rn": patch
+"jazz-tools": patch
+---
+
+## 💥 Breaking Changes
+
+**Removed `RNQuickCrypto` in favor of `RNCrypto`**
+
+`RNQuickCrypto` has been removed to streamline the library. `RNCrypto` is now the default implementation.
+
+**Migration Guide:**
+
+If you are currently using `RNCrypto`, you need to update your import paths:
+
+**For React Native:**
+```diff
+- import { RNCrypto } from "jazz-tools/react-native-core/crypto/RNCrypto";
++ import { RNCrypto } from "jazz-tools/react-native-core/crypto";
+}
+```
+**For Expo:**
+```diff
+-  import { RNCrypto } from "jazz-tools/react-native-core/crypto/RNCrypto";
++  import { RNCrypto } from "jazz-tools/expo/crypto";
+}
+```
+
+If you are not currently using `RNCrypto`, please refer to the [setup guide](https://jazz.tools/docs/react-native-expo/project-setup/providers#rncrypto).
```

Comment:
This part of the docs should go away, and we should make the installation of cojson-core-rn part of the installation guide